### PR TITLE
implement the help modal for Capidle

### LINKED ISSUE #63: help dialogue for Capidle
A help dialogue which explains how the game works would be very useful for new users. 

Related to #18

### PR DESCRIPTION
**Pull request by k-yle**
Closes #63 

To test: go to the Capidle mini game, and the help modal will appear. 

I also made some other changes to the modal while working on this PR:
 - merged two javascript files for the modal 
 - made the modal fade in/out
 - allow the modal to be closed using the <kbd>Esc</kbd> key, [for accessbility](http://simplyaccessible.com/article/closing-modals/)

### CODE REVIEW

File: src/capidle/index.html [hunk 1]
```diff
@@ -19,9 +19,51 @@
...
+            <span>↙️</span>
+          </div>
+        </div>
+      </div>
+      <p class="modal-text">
+        The hint tells you the how far away you are from the correct location,
+        and in what cardinal direction.
+      </p>
+      <p class="modal-text">
+        ❤️ The aerial imagery is sourced from
```
Comment (Destiny-02) [inline src/capidle/index.html:46]: Are you able to move the attribution into the `README` so it's all in one place? I don't think it belongs in the help dialogue which we want to keep as simple and clutter-free as possible.

File: src/capidle/index.html [hunk 2]
```diff
@@ -19,9 +19,51 @@
...
+        <div class="guess-rows" style="max-width: 300px">
+          <div class="guess">
+            <span>Berlin</span>
+            <span>1,200km</span>
+            <span>↙️</span>
+          </div>
+        </div>
+      </div>
+      <p class="modal-text">
+        The hint tells you the how far away you are from the correct location,
```
Comment (Destiny-02) [inline src/capidle/index.html:42]: Maybe you could add a line break here to make the dialogue a bit less wide

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
The fade is a nice little addition to the modal window :). Left some comments about the window design. Btw `navigation.cy.js` is failing.

File: src/capidle/index.html (hunk 2 above)
Comment (k-yle) [reply src/capidle/index.html]: this was a bit tricky so I fiddled with the css in 3c92164b

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
LGTM on the changes but I think `help-modal.cy.js` belongs with the homepage test files since its behaviour was defined in `modal.js` which is in the same directory as the homepage files. If `help-modal.cy.js` is moved there, then the `describe('help window' ...` tests should be removed from `start.cy.js`. (Sorry I probably should have done this in PR #76.) I also think the homepage files and tests should go in a separate subfolder but that can be for a future PR.

**Comment by k-yle** [comment]
good point, I've made that change

**Review by Destiny-02 — APPROVED** [review]
LGTM

**Pull request merged**